README.md

```diff
@@ -37,8 +37,6 @@ Version matrix:
 | 1.8.X              | v0.2              | release-1.8      |
 | 1.7.X              | v0.1              | release-1.7      |
 
-See the [roadmap](docs/roadmap.md) for information about current and future milestones.
```

Comment:
if you are deleting the link, may also delete the roadmap itself.

Would be great if you can help review the roadmap document and post what was NOT implemented out of it. It may be better to track those as issues in the repository.